tests/src/test/java/org/jboss/windup/tests/bootstrap/migrate/ExportCsvTest.java

```diff
@@ -23,11 +23,15 @@ public void exportCsv() throws IOException {
                 "--exportCSV");
 
         File csv = new File(tmp.getRoot(), "Windup1x_javaee_example_tiny_war.csv");
-
         assertTrue(csv.exists());
 
         String csvContent = new String(Files.readAllBytes(csv.toPath()), "UTF-8");
-
         assertTrue(csvContent.contains("Windup1x-javaee-example-tiny.war"));
+
+        File allIssuesCsv = new File(tmp.getRoot(), "AllIssues.csv");
+        assertTrue(allIssuesCsv.exists());
+
+        File appTagsCsv = new File(tmp.getRoot(), "ApplicationFileTechnologies.csv");
+        assertTrue(appTagsCsv.exists());
```

Comment:
Probably I haven't understood the problem....but, was not the goal of the ticket to provide a sort on the tags in the CSV file generated ? where are we checking that the content of the CSV file is following that ?